eZ/Publish/API/Repository/ContentService.php

```diff
@@ -254,10 +265,12 @@ public function createContentDraft(ContentInfo $contentInfo, VersionInfo $versio
      * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException if the current-user is not allowed to load the draft list
      *
      * @param \eZ\Publish\API\Repository\Values\User\User $user The user to load drafts from if defined, otherwise drafts for current-user
+     * @param int $limit
+     * @param int $offset
      *
      * @return \eZ\Publish\API\Repository\Values\Content\VersionInfo[] the drafts ({@link VersionInfo}) owned by the given user
      */
-    public function loadContentDrafts(User $user = null);
+    public function loadContentDrafts(User $user = null, int $offset = 0, int $limit = -1);
```

Comment:
What about sorting and filtering ? From PAPI perspective proposed change is fine, but ezplatform-admin-ui requirements still cannot be meet without loading full list of record set. Ref. 

* https://github.com/ezsystems/ezplatform-admin-ui/blob/55f5c6fe5f46cbf177b5ced48237a21605839232/src/lib/UI/Dataset/ContentDraftsDataset.php#L79-L83
* https://github.com/ezsystems/ezplatform-admin-ui/blob/55f5c6fe5f46cbf177b5ced48237a21605839232/src/lib/UI/Dataset/ContentDraftsDataset.php#L65-L68